<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">#frame{border-image: url(:/images/0.png);}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>110</y>
        <width>951</width>
        <height>73</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 32pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Hash与二分查找统计C关键字频度</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_select">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>260</y>
        <width>341</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 26pt &quot;Comic Sans MS&quot;;
background-color: rgb(0, 255, 255);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>选择C源文件</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_start">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>540</y>
        <width>251</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 32pt &quot;Comic Sans MS&quot;;
background-color: rgb(0, 255, 255);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>260</y>
        <width>901</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 18pt &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
